heltour/api_worker/views.py

```diff
@@ -35,6 +35,16 @@ def _do_lichess_api_call(redis_key, path, method, post_data, params, priority, m
         else:
             r = requests.get(url, params, headers=headers)
 
+        if r.status_code == 429:
```

Comment:
This is supposed to be handled below (line 61 of the original file)

Reply:
:+1:

Reply:
Done.